src/style.css

```diff
@@ -0,0 +1,137 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --main-color: #060b35;
+  --Secondary-color: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  background-color: var(--white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__description {
+  display: flex;
+  flex-direction: column;
+  margin-top: 40px;
+}
+
+.card__description-title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__description-code {
+  padding-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--Secondary-color);
+}
+
+.card__review-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.card__review {
+  align-self: center;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.card__price-cost {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__price-name {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--Secondary-color);
+}
+
+.card__button {
+  width: 100%;
+  display: flex;
+  margin-top: 16px;
+  text-align: center;
+  justify-content: center;
+}
+
+.card__button-link {
+  align-items: center;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: var(--white);
+  background-color: var(--blue-accent);
+  border-radius: 5px;
+  padding: 12px 69px;
+  border: 1px solid var(--blue-accent);
+  text-decoration: none;
+}
+
+.card__button:hover .card__button-link:hover {
+  background-color: var(--white);
+  color: var(--blue-accent);
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
create a separate file for each BEM block

Reply:
@sTorba24 done